src/main/java/reposense/model/ReportConfiguration.java

```diff
@@ -0,0 +1,20 @@
+package reposense.model;
+
+/**
+ * Represents configuration information from JSON config file for generated report.
+ */
+public class ReportConfiguration {
+    private static final String DEFAULT_TITLE = "RepoSense Report";
+    private String title;
+
+    public ReportConfiguration() {
+    }
+
+    public String getTitle() {
+        if (title == null) {
+            return DEFAULT_TITLE;
```

Comment:
Would it be more ideal if we set the `DEFAULT_TITLE` in the constructor? The title variable can then be overridden when parsing the configuration file.

Reply:
I'm ok with this design as it is. 🤔 If there is a consensus that setting the default title in the constructor is better, then lets change it.